packages/common-sdk/src/web3/transactions/types.ts

```diff
@@ -22,12 +28,14 @@ export type Instruction = {
  * @category Transactions Util
  */
 export type TransactionPayload = {
-  transaction: Transaction;
+  transaction: Transaction | VersionedTransaction;
```

Comment:
I did not execute the code, so if it is not breaking, please ignore it.🙏

is there small breaking ?
This does not seem to be so much of a problem since buildAndExecute() is used in many cases, but I feel it would be better to make it explicit in Breaking.

```
const payload = txbuilder.build();
payload.transaction....
```

should be modified to

```
const payload = txbuilder.build();
const transaction = payload.transaction as Transaction;
transaction....
```

Reply:
hmm. yeah good point. It will break if someone expects payload.transaction to be Transaction w.o type-casting. I will add it to the notes.